content/developer/writing_widgets/widget-definition.md

```diff
@@ -5,6 +5,7 @@ description: Description of widget definition including all available configurat
 category: Cloudify Console
 draft: false
 weight: 300
+ui_components_link: "https://docs.cloudify.co/ui-components/2.2.0"
```

Comment:
We're going back and forth here :smile: IMO that's now redundant since we have a link to the _Widgets components_ section in the docs that has a link to the most recent version of ui-components.

Thus, to not duplicate the information and keep the link up-to-date, I think we should remove the hardcoded link to ui-components from this page and let the user get the latest one by going to _widgets components_ first :slightly_smiling_face:

Reply:
fixed

Reply:
Hold on a second... What did you fix? I still see the link to the UI components hardcoded here